spec/customer_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'customer'
+
+describe Customer do
+
+  let(:takeaway) { Takeaway.new }
+
+    it 'customer should be able to view menu' do
+      expect(subject).to respond_to(:view_menu).with(1).argument
+    end
+
+    it 'menu should appear when view_menu is called' do
+      expect(subject.view_menu(takeaway)).to eql takeaway.menu
+    end
+
+  describe '#add_item' do
+
+    it 'customer should be able to add_item with 2 arguments' do
+      expect(subject).to respond_to(:add_item).with(2).argument
+    end
+
+    it 'each item passed to method should be placed in basket' do
+      subject.add_item(takeaway, "Steak and chips")
```

Comment:
Since you're testing `add_item` with 2 arguments here, you don't need the test in 17 - 19. Avoid duplicating the stuff you're testing.